cypress/e2e/authenticatedFlows/acceptable_use.ts

```diff
@@ -0,0 +1,50 @@
+const AcceptableUsePage = () => {
+  describe("Test acceptable use Page", () => {
+    beforeEach(() => {
+      cy.userSession({ admin: false, acceptableUse: false });
+      cy.visitPage("/en/auth/policy");
+    });
+
+    // The calls to api/session are causing the Accept button to become in a detached state when trying to click on it.
```

Comment:
i think this comment is not relevant anymore?

Reply:
yup, no longer relevant